integration_tests/src/utils.rs

```diff
@@ -63,6 +64,9 @@ pub const STAKING_TOKEN: &str = "aalthea";
 pub const MIN_GLOBAL_FEE_AMOUNT: u128 = 10;
 pub const TOTAL_TIMEOUT: Duration = Duration::from_secs(300);
 pub const DEBT_ACCURACY_THRES: u8 = 15;
+pub const ETH_NODE: &str = "http://localhost:8545";
+pub const MINER_PRIVATE_KEY: &str =
```

Comment:
instead of parsing this every time everywhere just having a function that does it and returns the actual PrivateKey struct will probably save us a lot of duplicated code.

Reply:
done